angular/projects/common/src/app/legislation/legislation-list-detail-public/legislation-list-detail.component.spec.ts

```diff
@@ -32,4 +33,13 @@ describe('LegislationListDetailComponent', () => {
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+
+  it('should display an actCode when the backend cannot return the legislation code map', async () => {
+    const testActCode = 'ACT_103';
```

Comment:
[Arrange, Act, Assert](https://automationpanda.com/2020/07/07/arrange-act-assert-a-pattern-for-writing-good-tests/) can be a good pattern to follow when writing tests to improve clarity.

Reply:
reformatted to follow this pattern